js/filteringBox.js

```diff
@@ -27,6 +27,78 @@ function openFilterModal() {
     closeFilterButton.classList.add("filter__box__topDiv__closeFilterBtn");
     closeFilterButton.innerHTML = "X";
     closeFilterButton.addEventListener("click", closeFilterModal);
+
+    // Checkbox divs
+    const checkDivMain = document.createElement("div");
+    filterBox.appendChild(checkDivMain);
+    checkDivMain.classList.add("filter__box__checkDivMain");
+
+    const checkDiv2 = document.createElement("div");
+    checkDivMain.appendChild(checkDiv2);
+    checkDiv2.classList.add("filter__box__checkDivMain__checkDiv2");
+
+    const checkDiv3 = document.createElement("div");
+    checkDivMain.appendChild(checkDiv3);
+    checkDiv3.classList.add("filter__box__checkDivMain__checkDiv3");
+
+    const checkDiv4 = document.createElement("div");
+    checkDivMain.appendChild(checkDiv4);
+    checkDiv4.classList.add("filter__box__checkDivMain__checkDiv4");
+
+    // Checkboxes
+    const checkBoxTitle = document.createElement("h4");
```

Comment:
One more point :)
Our project has a semantic structure.
I notice that you have an <h2> heading followed immediately by <h4>
This might play a small role in how the text is read by people with disabilities.